runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java

```diff
@@ -376,7 +376,7 @@ static BufferedImage fillImage(final BufferedImage image, final Color color)
 	 * @return              The given image with all pixels fulfilling the fill condition predicate
 	 *                      set to the given color.
 	 */
-	static BufferedImage fillImage(final BufferedImage image, final Color color, final Predicate<Color> fillCondition)
+	public static BufferedImage fillImage(final BufferedImage image, final Color color, final Predicate<Color> fillCondition)
```

Comment:
I think this one can still remain package-private? It doesn't look like it's being called.

Reply:
good catch. Done